### ISSUE TITLE
False-positive by indent rule

### ISSUE THREAD

**Issue by pascalmr** [yoheimuta/protolint#74]
Maybe related to fixing #66 I encountered a false-positive of the indent rule in the newest version 0.16.0 which is reported if the last rpc method of a service is having a statement block:
```
service SearchService {
  rpc Search (SearchRequest) returns (SearchResponse) {}
} // -> Found an incorrect indentation style "". "  " is correct.
```
In my case the rpc statement block is containing an option, but it's not needed to get the issue.

**Comment by yoheimuta** [comment]
@pascalmr Thank you for your report!
You are right. I can reproduce this issue when protolint depends on the new go-protoparser.

I will fix it.

**Comment by yoheimuta** [comment]
I fixed it by https://github.com/yoheimuta/protolint/pull/75.
I have cut a release as v0.16.1.

@pascalmr Please let me know whenever you can still experience the problem.
I appreciate your detailed report. Thank you.
